### PR TITLE
Added mapGL recipe.

### PR DESCRIPTION
**Pull request by adadiehl**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/mapgl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - mapGL.py --help
+
+about:
+  home: "https://github.com/adadiehl/mapGL"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: ""
+  summary: "Prediction of lineage-specific gain and loss of sequence elements using phylogenetic maximum parsimony."
+  doc_url: ""
+  dev_url: ""
```
Comment (dpryan79) [inline recipes/mapgl/meta.yaml]: Can you remove the empty doc_url and dev_url entries? If there's no license that comes with the code then remove the license_file entry too.

File: recipes/mapgl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  entry_points:
+    - mapGL.py = map_GL.mapGL:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: generic
+
+requirements:
+  host:
+    - pathlib
+    - bx-python
+    - numpy
```
Comment (dpryan79) [inline recipes/mapgl/meta.yaml]: Does it really need, pathlib only during installation? Does it link against numpy or bx-python? If not, please remove them from `host:`.

File: recipes/mapgl/meta.yaml (hunk 1 above)
Comment (adadiehl) [reply recipes/mapgl/meta.yaml]: Fixed.

File: recipes/mapgl/meta.yaml (hunk 2 above)
Comment (adadiehl) [reply recipes/mapgl/meta.yaml]: Retrying without pathlib.

**Comment by dpryan79** [comment]
I'll hit merge on this later tonight once the OSX test has a chance to run. Thanks!

**Pull request merged**